src/lobos/schema.clj

```diff
@@ -689,15 +690,15 @@
   [name & [columns constraints indexes]]
   (name-required name "table")
   (Table. name
-          (or columns {})
+          (or columns (ordered-map))
```

Comment:
I think it's better to order the whole expression : 

```clojure
(ordered-map (or columns {}))
```

Beause there is other places in code where columns should be ordered. See `defmethod analyze [::standard Table]` in `analyzer.clj`